README.md

```diff
@@ -7,3 +7,42 @@ Most of the bindings have been implemented, though most haven't been verified,
 yet. Please see the `samples` directory for examples, ported from the tutorials
 found at <http://lazyfoo.net/tutorials/SDL/index.php>. You are welcome to port
 more samples, and the necessary corrections!
+
+## Requirements
```

Comment:
Please drop the library requirements. Only `libSDL2` by-the-way, other libraries are optional.

Also drop the homebrew commands; SDL2 is multiplatform and we shouldn't have to tell how to install the libraries. One may choose the solution that best fits them (distrib packages, manual compilations, ...).